components/script/stylesheet_loader.rs

```diff
@@ -147,6 +150,9 @@ impl FetchResponseListener for StylesheetContext {
                 }
                 StylesheetContextSource::Import(ref import) => {
                     let import = import.read();
+                    if elem.downcast::<HTMLLinkElement>().unwrap().is_alternate() {
```

Comment:
My mistake, that test doesn't include any stylesheets. Try `./mach test-wpt tests/wpt/mozilla/tests/mozilla/stylesheet-adopt-panic.html` instead.

My reading of the spec doesn't include any way of specifying a style element as an alternate stylesheet. Where did you see that?